clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_2_0.xsd

```diff
@@ -287,7 +287,7 @@
     </xs:complexType>
 
     <xs:complexType name="locking">
-        <xs:attribute name="isolation" type="tns:isolation" default="REPEATABLE_READ">
+        <xs:attribute name="isolation" type="tns:isolation" default="READ_COMMITTED">
```

Comment:
Shouldn't the parser be adjusted to handle default values correctly?

Yes - we should fix the parsers, but only since 3.0.  Prior to 3.0, the default really was REPEATABLE_READ.